### PR TITLE
feat: Fixes for product editor

### LINKED ISSUE #6114: Missing fields on Product GQL schema and more translation strings needed
## Issue Description

This issue is resolving missing things found when updating the product admin UI to use the GQL API. https://github.com/reactioncommerce/reaction-admin/pull/188

Product schema is missing fields for
- `shouldAppearInSitemap`

Need additional translations added to API
- "Create a variant"
- "Create a variant option"
- "Make visible in storefront"
- "Delete this variant"
- "Delete this option"

### PR DESCRIPTION
**Pull request by mikemurray**
Resolves #6114 
Impact: **minor**
Type: **feature|bugfix**

## Issue

Fix various issues found when working on the new product editor: https://github.com/reactioncommerce/reaction-admin/pull/188

## Solution

Add translations and missing fields in schemas

## Breaking changes

None.

## Testing
1. Test with: https://github.com/reactioncommerce/reaction-admin/pull/188
2. Ensure that translations look good
3. On the product detail form, check/uncheck "show in sitemap" checkbox and save.

### CODE REVIEW

File: src/core-services/product/mutations/updateProduct.js [hunk 1]
```diff
@@ -92,6 +92,10 @@ const inputSchema = new SimpleSchema({
     type: String,
     optional: true
   },
+  "shouldAppearInSitemap": {
```
Comment (spencern) [inline src/core-services/product/mutations/updateProduct.js:95]: Do we have any patterns that we follow for how we name boolean variables? 
The word `should` here feels a little ambiguous.

Comment (spencern) [reply src/core-services/product/mutations/updateProduct.js:95]: To expand on this, we use `isDeleted` and `isVisible` above. Would this variable be better as `isInSitemap`?

Comment (mikemurray) [reply src/core-services/product/mutations/updateProduct.js:95]: we do and this is one of them.

Comment (mikemurray) [reply src/core-services/product/mutations/updateProduct.js:95]: This is an existing property of the product schema. It wasn't exposed in GraphQL queries/mutations. "should" is allowed for boolean values, along with "is". This convention seem to be inforced for React components. I don't always see it inforced for other types of JS.

Comment (willopez) [reply src/core-services/product/mutations/updateProduct.js:95]: this seems ok to me

**Review by willopez — APPROVED** [review]
tested with corresponding Reaction Admin UI and seems good.

**Pull request merged**